researcher.lic

```diff
@@ -0,0 +1,183 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#check-symbiosis
+=end
+begin
+  custom_require.call %[common common-arcana drinfomon events]
+rescue
+  echo "Dependency is not running! This script requires dependency to be installed and running."
+  exit
+end
+
+
+class Researcher
+  include DRC
```

Comment:
I know this is how the script was written originally, now that we're bringing it in then we're trying to refactor code to not use `include` statements. 

Please remove the include and then add the 'DRC.` prefix in the file for methods like bput or message or anything else that comes from common.lic

Thanks!